lib/src/Base/Func/SpecFunc/openturns/SpecFunc.hxx

```diff
@@ -104,6 +104,8 @@ public:
   // Maximum positive real number
   static const Scalar MaxScalar;
   static const Scalar LogMaxScalar;
+  // Minimum negative real number;
+  static const Scalar LowestScalar;
```

Comment:
what's the point of adding it if its not used ?

Reply:
It's going to be used! I'll change the early commits during rebase in order to use it.

Reply:
oh, sorry